docs/core/porting/third-party-deps.md

```diff
@@ -44,14 +46,17 @@ netstandard1.3
 netstandard1.4
 netstandard1.5
 netstandard1.6
+netstandard2.0
 netcoreapp1.0
+netcoreapp1.1
+netcoreapp2.0
 portable-net45-win8
 portable-win8-wpa8
 portable-net451-win81
 portable-net45-win8-wpa8-wpa81
 ```
 
-These are the Target Framework Monikers (TFM) which map to versions of the [.NET Standard](../../standard/net-standard.md) and traditional Portable Class Library (PCL) profiles which are compatible with .NET Core.  Note that `netcoreapp1.0`, while compatible, is for applications and not libraries.  Although there's nothing wrong with using a library which is `netcoreapp1.0`-based, that library may not be intended for anything *other* than consumption by other `netcoreapp1.0` applications.
+These values are the [Target Framework Monikers (TFMs)](../../standard/frameworks.md) that map to versions of the [.NET Standard](../../standard/net-standard.md), .NET Core, and traditional Portable Class Library (PCL) profiles that are compatible with .NET Core. Note that `netcoreapp`, while compatible, is for .NET Core projects only and not for .NET Standard. Although there's nothing wrong with using a .NET Core library (`netcoreapp`-based), that library can only be consumed by other .NET Core apps.
```

Comment:
The last two sentences in this paragraph are really important for customers to understand. Is there a way to make them stand out more? Make them their own paragraph, or highlight them in some other way?

Reply:
Let me know what you think of the changes @BillWagner